src/utils.jl

```diff
@@ -21,31 +19,19 @@ Zygote.@nograd function generate_cluster(M::AbstractArray{T,N}, accu_edge) where
 end
 
 """
-    edge_index_table(adj[, directed])
+    edge_index_table(adj)
 
 Generate a mapping from vertex pair (i, j) to edge index. The edge indecies are determined by
 the sorted vertex indecies.
 """
-function edge_index_table(adj::AbstractVector{<:AbstractVector{<:Integer}}, directed::Bool=is_directed(adj))
+function edge_index_table(adj::AbstractVector{<:AbstractVector{<:Integer}})
```

Comment:
I suppose to move these utilities to GraphSignals.jl.

Reply:
edge_index_table won't be needed anymore once other pieces of #204 are merged